README.md

```diff
@@ -21,6 +22,9 @@ Colo Loco will find your colocated native files, automatically link them up to t
 
 ### Watch Jamon show how to use React Native Colo Loco!
 
+[Watch the tutorial on YouTube](https://www.youtube.com/watch?v=gfORZXq4ZgE)
+
+In this video, Jamon demonstrates how to use React Native Colo Loco by hacking Xcode workspace files. He also introduces a new feature requested by one of his project teams at Infinite Red and shows how to do test-driven development the easy way with ChatGPT and Github Copilot. Watch the video to learn more about using React Native Colo Loco!
```

Comment:
I don't actually show how to _use_ RN Colo Loco, but rather just how to contribute to it.